internal/debug/debug.go

```diff
@@ -0,0 +1,24 @@
+package debug
+
+import (
+	"log"
+	"os"
+	"strconv"
+)
+
+const (
+	EnvVar = "PAT_DEBUG"
+	Prefix = "[DEBUG] "
+)
+
+func enabled() bool {
+	b, _ := strconv.ParseBool(os.Getenv(EnvVar))
```

Comment:
This probably isn't that expensive, but it's potentially called many times. Is it worth initializing a variable for this?

Reply:
I agree 👍  No use in supporting env variable changes while the application is running.

I've fixed it by parsing the env variable in func init() 🙂